drake/systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -285,27 +301,30 @@ function checkDirty(obj)
         obj.LCP_cache.data.u = u;
         obj.LCP_cache.data.nargout = nargout;
 
-        num_q = obj.manip.num_positions;
-        q=x(1:num_q); qd=x(num_q+(1:num_q));
+        num_q = obj.manip.getNumPositions;
+        num_v = obj.manip.getNumVelocities;
+        q=x(1:num_q); v=x(num_q+(1:num_v));
+        vToqdot = obj.manip.vToqdot(q);
```

Comment:
does this take care of the normalization for us?  or is it happening somewhere else i missed?